tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -875,6 +875,11 @@ def setUp(self):
                 # make sure we restart to avoid leaving us in a bad state
                 self.device.start_b2g()
 
+        # Make sure the screen isn't turned off in lockscreen mode
+        self.marionette.switch_to_frame()
```

Comment:
I think we're already in this context, but leaving this in to be sure is fine with me.